.github/workflows/ci.yml

```
@@ -17,10 +17,37 @@ env:
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION_2 }}
+
+      - name: Install dotnet-format
+        run: dotnet tool install --global dotnet-format
+      - name: Run formatter for lib
+        run: dotnet format ${{ env.PROJECT_PATH }} --verify-no-changes 
+      - name: Run formatter for sample Console
+        run: dotnet format ${{ env.SAMPLE_CONSOLE_PATH }} --verify-no-changes 
+      - name: Run formatter for sample StreamPlay
+        run: dotnet format ${{ env.SAMPLE_CONSOLE_PATH }} --verify-no-changes 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)